src/extension.ts

```diff
@@ -136,6 +158,13 @@ function fetchAndLaunchMetals(context: ExtensionContext, javaHome: string) {
       ? {}
       : { COURSIER_REPOSITORIES: customRepositories };
 
+  const customRepositoriesParams = new Array<string>();
+  // config
```

Comment:
I don't think this should be commented out and the same below.

Reply:
Fixed.